test/e2e/features/cat-d/01-pass-journy.feature

```diff
@@ -0,0 +1,36 @@
+@catd
```

Comment:
typo in file name 'journy' should be 'journey'

Reply:
@Cullima  Thank you typo error has been fixed.